comfy_cli/command/models/models.py

```diff
@@ -394,6 +394,11 @@ def remove(
         typer.echo("Deletion canceled.")
 
 
+def list_models(path: pathlib.Path) -> list:
```

Comment:
No functional change here, but if we were to redefine the built-in `list`, we'll need to make sure that any possible references to `list` should happen before the `def list()` function.